nextjs/src/modules/newsletter/IndividualNewsletterPage.tsx

```diff
@@ -15,7 +14,16 @@ import { NewsletterPagination } from './components/NewsletterPagination';
 /** -------------------------------------------------
 * COMPONENT
 ---------------------------------------------------- */
-const IndividualNewsletterPage = ({ dateSent, subject, content, meta, pagination }) => {
+
+interface Props {
+  dateSent: string;
+  subject: string;
+  content: string[];
+  meta: {};
+  pagination: {};
+}
```

Comment:
not 100% sure these types are correct as I did not know the shape of meta and pagination